### DISCUSSION TITLE
Shortcut Maker

### DISCUSSION THREAD

**Discussion by Sha23di** [akaMrNagar/Mindful#20] [General]
How to make a shortcut using Shortcut Maker app to Start Focus Session immediately?

**Comment by akaMrNagar** [comment]
Will let you know after testing!!

**Comment by akaMrNagar** [comment]
So the shortcut maker uses activities to target screen in app but Mindful is build with flutter and does not uses multi-activity architecture. Thus making it impossible for other apps to open any screen on Mindful. 

The conclusion is you cannot make shortcut for focus screen. I have some plans regarding this issue but you have to wait for a while.

**Comment by akaMrNagar** [comment]
I have added deep linking to Mindful for this purpose and the documentation is at https://github.com/akaMrNagar/Mindful/blob/main/docs/API.md 

Use this to open any screen in Mindful. I have also added Quick Settings Focus Tile and some app shortcuts in the latest release
